tests/unit/test_session.py

```diff
@@ -697,7 +697,10 @@ def test_create_client_verify_param_overrides_all(self, client_creator):
             self.assertEqual(call_kwargs['verify'], 'verify-certs.pem')
 
     @mock.patch('botocore.client.ClientCreator')
-    def test_create_client_use_no_api_version_by_default(self, client_creator):
+    def test_create_client_use_no_api_version_by_default(
```

Comment:
Why did this file change? Was this just never run through the formatter before? We should make sure we are not making updates if there is no changes to the actual logic in order to help cut down the noise in reviews.

Reply:
From previous refactors this line was touched and then `pre-commit` cleaned it up. Will revert it.